Source/WebCore/platform/graphics/displaylists/DisplayListItems.h

```diff
@@ -495,7 +495,28 @@ class DrawGlyphs {
 public:
     static constexpr char name[] = "draw-glyphs";
 
-    DrawGlyphs(Ref<const Font>&& font, Vector<GlyphBufferGlyph>&& glyphs, Vector<GlyphBufferAdvance>&& advances, const FloatPoint& localAnchor,  FontSmoothingMode smoothingMode)
+#if USE(SKIA)
+    DrawGlyphs(Ref<const Font>&& font, Vector<GlyphBufferGlyph>&& glyphs, Vector<GlyphBufferAdvance>&& advances, const FloatPoint& localAnchor, FontSmoothingMode smoothingMode)
+        : m_localAnchor(localAnchor)
+        , m_fontSmoothingMode(smoothingMode)
+    {
+        m_textBlob = font->buildTextBlob(glyphs, advances, smoothingMode, m_enableAntialiasing, m_isVertical);
```

Comment:
This can go into the initializer list as well.

Reply:
Turns out we can't do that. My guess is that m_enableAntialiasing and m_isVertical are first set by buildTextBlob and after that set to their default value because of they are after m_textBlob in the class definition.